app/controllers/collections_controller.rb

```diff
@@ -11,7 +11,7 @@ def show
       collection = get_collection_from_record(@record)
 
       @concept_id = collection.concept_id
-      @collection_records = collection.get_records.order(:revision_id).reverse_order
```

Comment:
This logic will not properly order based on revision id since revision_id is a string, it will do a string comparison.